core/lib/network-request.js

```diff
@@ -184,6 +195,50 @@ class NetworkRequest {
     this.isLinkPreload = false;
   }
 
+  // TODO: remove after timing refactor is done
+  // See https://github.com/GoogleChrome/lighthouse/pull/14311
+  /**
```

Comment:
Easier just to `s/startTime/networkRequestTime/g`, `s/endTime/networkEndTime/g` and `s/responseReceivedTime/responseHeadersEndTime/g`? Seems unnecessary to have the codebase use both (and necessitates the `network-records-to-devtools-log.js` workaround) when there's very little review work for just a property rename.

Reply:
The big reason for this approach is that this PR is just 7 files (less, if I take your suggestion to defer network requests audit changes)- but it would be dozens of files mixed in with the important changes here if done all at once.

The few lines in records to DevTools log is just encoding the framework we'd mostly use during the conversion process (next PR).

Reply:
I was probably too focused on an exact approach in my suggestion :) I think we're all broadly going for the same thing, my main issue was just anticipating a PR where some `startTime`s get changed to `networkRequestTime` and some get changed to `rendererStartTime` (and similar for `endTime`) and a reviewer will have to read each line carefully to double check which one was used.

I was attempting to suggest
- a PR that only updates `startTime` to `networkRequestTime` (and `endTime` to  `networkEndTime`) so no actual timing/test values changed, and then
- a PR that changes some of those `networkRequestTime`s to `rendererStartTime` (and `networkEndTime` to `rendererEndTime`), which would involve some timing/test value changes. That way the changes we want to be most deliberate about are highlighted (any property being changed is switched to the new timing value).

If intermediate PRs before that point help break it up nicely, SGTM, I won't object to that :)

Reply:
> a PR that only updates startTime to networkRequestTime (and endTime to networkEndTime) so no actual timing/test values changed

Doing this next makes sense for the reasons you highlighted. Will do that. Hopefully it'll be the fasted on these PRs!